awx/main/tasks/jobs.py

```diff
@@ -948,6 +945,10 @@ def build_env(self, job, private_data_dir, private_data_files=None):
             paths = [os.path.join(CONTAINER_ROOT, folder)] + paths
             env[env_key] = os.pathsep.join(paths)
 
+        env['ANSIBLE_CALLBACKS_ENABLED'] = 'indirect_instance_count'
+        if 'callbacks_enabled' in config_values:
+            env['ANSIBLE_CALLBACKS_ENABLED'] += ':' + config_values['callbacks_enabled']
```

Comment:
@jessicamack around here, in particular, we would like to check the feature flag to tell if the indirect host counting is enabled. How do I do that?

Reply:
Here is an example: https://github.com/ansible/awx/pull/15750/files#diff-d02e710a09a5ab2feac829d66de9c38fd74122975de65f763cd62ecc1f3e77b7R26